app/config/config.yml

```diff
@@ -87,8 +84,13 @@ doctrine_migrations:
             executed_at_column_name: 'executed_at'
 
 fos_rest:
+    zone:
+        - { path: ^/api }
+        - { path: ^/annotations }
     param_fetcher_listener: true
     body_listener: true
+    exception:
+        serializer_error_renderer: true
```

Comment:
this is needed so that exceptions thrown manually in rest controllers like in https://github.com/wallabag/wallabag/blob/master/src/Wallabag/ApiBundle/Controller/EntryRestController.php#L313 respond with JSON instead of HTML